src/wallet/rpcdump.cpp

```diff
@@ -183,7 +183,7 @@ UniValue importaddress(const UniValue& params, bool fHelp)
         std::vector<unsigned char> data(ParseHex(params[0].get_str()));
         script = CScript(data.begin(), data.end());
     } else {
-        throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Invalid Bitcoin address or script");
+        throw JSONRPCError(RPC_NOT_FOUND, "Invalid Bitcoin address or script");
```

Comment:
Not sure about that. This example (rpcdum.cppL186): there `RPC_INVALID_ADDRESS_OR_KEY` is totally fine IMO.